source/includes/aws/_nodegroups.md

```diff
@@ -0,0 +1,118 @@
+### Nodegroups
+<!-------------------- RETRIEVE NODEGROUPS -------------------->
+
+#### List nodegroups
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v2/services/aws/test-area/nodegroups"
```

Comment:
Doesn't it requires the cluster id as a parameter?

Reply:
I modified the nodegroup fetcher to not require a cluster ID to fetch all of the nodegroups of all clusters, to me this makes sense since for fetching the list of all subnetworks, you don't have to use the vpc ID as a parameter, what do you think?

Reply:
maybe work mentioning that you can add the clusterid as a parameter to have the list for that cluster alone

Reply:
Added info regarding that 👌🏻